lib/back.js

```diff
@@ -245,16 +245,19 @@ function fixtureExists(fixture) {
 }
 
 function assertScopes(scopes, fixture) {
-  scopes.forEach(function(scope) {
-    expect(scope.isDone()).to.be.equal(
-      true,
+  const pending = scopes
+    .filter(scope => !scope.isDone())
+    .map(scope => scope.pendingMocks())
+
+  if (pending.length) {
+    assert.fail(
       format(
         '%j was not used, consider removing %s to rerecord fixture',
-        scope.pendingMocks(),
+        [].concat(...pending),
```

Comment:
Does this look reasonable when there are more than one unfulfilled request?

Reply:
It makes it look the same whether those pending mocks are from one Scope or multiple. 
Previously, only the first Scope's worth would show in the message.

Reply:
It'd be nice to add a test of the concatenation, though it's not essential for merging this.